<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nombre cible</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"> QPushButton[flat=&quot;true&quot;]{ 
		color : red; 
		background-color : yellow; 
		border-radius : 8px; 
		border-width : 2px;
		border-style : solid ;
		border-color : black; 
}
QPushButton { 
	background-color: rgb(0, 170, 0);
		border-radius : 8px; 
		color: yellow;
		border-width : 2px;
		border-style : solid ;
		border-color : black;
}
QPushButton:hover{ 
		color : yellow;
		background-color :   rgb(0, 170, 0);
		border-radius : 8px; 
		border-width : 2px;
		border-style : solid ;
		border-color : white; 
}
QPushButton#btnQuitter { 
		background-image: url(:/img/data/images/quit-24x24.png);
		background-color: rgb(170, 170, 0);
		border-radius : 8px; 
		background-repeat: repeat-no;
   		background-position: center;
		border-width : 2px;
		border-style : solid ;
		border-color : black;
}
QComboBox:enabled {
	background-color: rgb(0, 170, 0);
		border-radius : 4px; 
		color: yellow;
		border-width : 2px;
		border-style : solid ;
		border-color : black;
}
QComboBox:!enabled {
		background-color: grey;
		border-radius : 4px; 
		color: black;
		border-width : 2px;
		border-style : solid ;
		border-color : black;
}
QPushButton#btnVerifier { 
	background-color: rgb(255, 170, 0);
	color: black;
} 
QPushButton#btnNouveau { 
	background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(20, 128, 0, 255), stop:1 rgba(197, 255, 238, 255));
		color: black;
} 
QLabel#lblAffiche {
		background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 255, 50, 255), stop:1 rgba(255, 255, 255, 255));
		border-radius : 8px; 
		background-image: url(:/img/data/images/pagefold-50x45.png);
		background-repeat: repeat-no;
   		background-position: top right;
}

QLabel#lblCible { 
		background-color: rgb(255, 170, 0);
		border-radius : 60px;
		border-width : 20px;
		border-style : solid ;
		border-color :  rgb(255, 70, 0);
}

QWidget#centralWidget { background-image: url(:/img/data/images/background-600x400.png); }
</string>
   </property>
   <widget class="QPushButton" name="btnNbre0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNbre1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNbre2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNbre3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNbre4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNbre5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNbre6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNbre7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNbre8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNbre9">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRep0">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>230</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRep1">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>230</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRep2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>230</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblCible">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNouveau">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>320</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Nouveau</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cboxJoker">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>?</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lblAffiche">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnQuitter">
    <property name="geometry">
     <rect>
      <x>517</x>
      <y>320</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuNiveau">
    <property name="title">
     <string>Niveau</string>
    </property>
    <addaction name="actionDebutant"/>
    <addaction name="actionJoker"/>
    <addaction name="actionSurcomptage"/>
    <addaction name="actionCalcul"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuNiveau"/>
  </widget>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionDebutant">
   <property name="text">
    <string>Débutant</string>
   </property>
  </action>
  <action name="actionJoker">
   <property name="text">
    <string>Avec joker</string>
   </property>
  </action>
  <action name="actionAvec_joker">
   <property name="text">
    <string>Avec joker</string>
   </property>
  </action>
  <action name="actionSurcomptage">
   <property name="text">
    <string>Surcomptage</string>
   </property>
  </action>
  <action name="actionCalcul">
   <property name="text">
    <string>Calcul</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
